.pre-commit-config.yaml

```diff
@@ -108,7 +108,7 @@ repos:
     hooks:
       - id: codespell
         additional_dependencies: ["tomli"]
-        args: [-L, lama]
+        args: ["-L", "lama", "--ignore-regex", ".{1024}|.*codespell-ignore.*"]
```

Comment:
This line allows us to add 'codespell-ignore' to any line to avoid codespell's check on that line. It also ignores any single line over 1024 chars in length, since those are almost always non-textual lines.